radio/src/pulses/pulses_arm.cpp

```diff
@@ -144,18 +149,22 @@ void setupPulses(unsigned int port)
   switch (required_protocol) {
     case PROTO_PXX:
       setupPulsesPXX(port);
+      SCHEDULE_MIXER_START(port, 9);
```

Comment:
Are you confident with this schedules? Could we use heartbeat signal from XJT module for scheduling?